geode-core/src/main/java/org/apache/geode/internal/cache/BucketRegion.java

```diff
@@ -582,6 +579,9 @@ public void cmnClearRegion(RegionEventImpl regionEvent, boolean cacheWrite, bool
       if (!isLockedAlready) {
         obtainWriteLocksForClear(regionEvent, participants);
       }
+      if (enableRVV) {
+        rvv = getVersionVector().getCloneForTransmission();
```

Comment:
Why is this needed. In Partitioned Region all the operation happens first on Primary and then secondary. And in case clear; other operations are blocked till clear is finished or clear is blocked till other operations are completed on both primary secondary.
By sending RVV; impacts the performance of clear operation; it will iterate one entry at a time and removes it.